### PR TITLE
KFSPTS-5490 Add active indicator for User Profile and Favorite Account

### PR DESCRIPTION
**Pull request by db483**
Hi Bryan,

Please review code changes related to KFSPTS-5490 to add active indicator on User Profile and Favorite Account. I have also assigned you the PR for the related SQL: https://github.com/CU-CommunityApps/nonprod-sql/pull/16

Please note I did tiny bit of cleaning with the changes too.

Please don't merge yet as I am still waiting for feedback from Cathy but wanted to get this out for review. (I don't expect significant changes after functional feedback)

### CODE REVIEW

**Comment by db483** [comment]
retest this please

File: src/main/resources/edu/cornell/kfs/sys/cu-ojb-sys.xml [hunk 1]
```diff
@@ -139,6 +139,7 @@
     <class-descriptor class="edu.cornell.kfs.sys.businessobject.UserProcurementProfile" table="CU_USER_PROCUREMENT_PROFILE_T">
     	<field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true"/>
     	<field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true"/>
+    	<field-descriptor name="active" column="ROW_ACTV_IND" jdbc-type="VARCHAR" conversion="org.kuali.rice.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion"/>
```
Comment (bhutchinson) [inline src/main/resources/edu/cornell/kfs/sys/cu-ojb-sys.xml]: I mentioned this in the sql review as well, but I think this should probably be just ACTV_IND (see: https://kualico.atlassian.net/wiki/x/Q4AZ).

File: src/main/resources/edu/cornell/kfs/sys/cu-ojb-sys.xml [hunk 2]
```diff
@@ -161,6 +162,7 @@
     	<field-descriptor name="projectCode" column="PROJECT_CD" jdbc-type="VARCHAR"/>
     	<field-descriptor name="organizationReferenceId" column="ORG_REF_ID" jdbc-type="VARCHAR"/>
     	<field-descriptor name="userProfileId" column="USER_PROFILE_ID" jdbc-type="INTEGER" />
+    	<field-descriptor name="active" column="ROW_ACTV_IND" jdbc-type="VARCHAR" conversion="org.kuali.rice.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion"/>
```
Comment (bhutchinson) [inline src/main/resources/edu/cornell/kfs/sys/cu-ojb-sys.xml]: ACTV_IND - see comment above.

**Comment by bhutchinson** [comment]
Looks good, just one minor thing that I commented on. Thanks for the cleanup.

**Comment by db483** [comment]
retest this please

**Comment by db483** [comment]
Hi Bryan, 
I have made the suggested change and this is ready for review again. Thank you.

**Comment by bhutchinson** [comment]
Changes look good and tests passed. This is ready to go, pending feedback from Cathy.

**Comment by db483** [comment]
Cathy approved the changes so we can merge this to develop.

**Pull request merged**